src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/activities/ActivitiesService.kt

```diff
@@ -140,7 +140,7 @@ class ActivitiesService(
         mappingService.getMappingGivenActivityScheduleId(allocation.scheduleId).let { mapping ->
           nomisApiService.createAllocation(
             mapping.nomisCourseActivityId,
-            CreateOffenderProgramProfileRequest(
+            CreateAllocationRequest(
```

Comment:
Also changing to NOT use the Nomis language for Allocation because it's so bad - see https://github.com/ministryofjustice/hmpps-nomis-prisoner-api/pull/255/files#r1165155902